packages/editor/src/state/atoms.ts

```diff
@@ -200,7 +200,10 @@ export const currentWorkspaceState = atom<Workspace>({
       },
       style: {
         name: ".style",
-        contents: "",
+        contents: `canvas {
+  width = 400
```

Comment:
is the spacing correct on these lines? they look like they're not indented enough in the PR